deployment/README.md

```diff
@@ -94,3 +94,66 @@ bash-4.4# ./scripts/infra apply
 ```
 
 This will attempt to apply the plan assembled in the previous step using Amazon's APIs.
+
+## Logs and Debugging
+
+
+### CloudWatch logs and Dashboard
+
+Navigate to [CloudWatch](https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#dashboards:). Select the dashboard for the environment you're investigating. Adjust the dashboard to reflect the time period you're interested in using the 'Custom' time filter, if needed.
+
+The Request Latency graph shows the top 50% fastest response times, the top 90%, and the top 99%. Requests that take over 30 seconds to respond will time out.
+
+The Requests graph shows the number of requests being received at a certain time.
+
+The Database CPU graph shows the percentage of database CPU being used at a certain time.
+
+CloudWatch logs are divided into groups. You will most likely be interested in the ones named log[Environment]App and log[Environment]AppCLI.
+
+CloudWatch also contains Log Insights, which allows you to query the CloudWatch logs (sample queries are available in AWS).
+
+### RDS
+
+Visit [RDS](https://eu-west-1.console.aws.amazon.com/rds/home?region=eu-west-1) in AWS.
+
+Navigate to [Databases](https://eu-west-1.console.aws.amazon.com/rds/home?region=eu-west-1#databases), then select the database you're investigating. Scroll down and navigate to 'Logs & events'.
+
+Identify the log(s) for the time period of interest (sorting by 'last written' will help). Note that the 'Last written' time displayed will be in your local time. Download the log or logs.
+
```

Comment:
Consider mentioning that logs can also be downloaded via the AWS CLI, which can be less cumbersome if you need to download a range of 1-hour log segments and can be automated. Here is an example command.

```
AWS_PROFILE=open-apparel-registry aws rds download-db-log-file-portion --db-instance-identifier  openapparelregistry-enc-prd  --log-file-name error/postgresql.log.2022-09-13-17 > postgresql.log.2022-09-13-17.txt
```

Reply:
Added